index.md

```diff
@@ -58,104 +58,105 @@ This page introduces guidelines and other standards related to web accessibility
 
-ATAG info:
-- [ATAG Overview](/standards-guidelines/atag/)
-- [[ATAG at a Glance]](/standards-guidelines/atag/glance/)
-- [ATAG 2.0 Standard](https://www.w3.org/TR/ATAG/)
+Información sobre las pautas UAAG:
+- [Resumen de UAAG](/standards-guidelines/uaag/)
+- [Nota sobre UAAG 2.0](https://www.w3.org/TR/UAAG20/)
 
-### User Agent Accessibility Guidelines (UAAG) {#uaag}
-
-User agents include browsers, browser extensions, media players, readers, and other applications that render web content.
-
-UAAG info:
-- [UAAG Overview](/standards-guidelines/uaag/)
-- [UAAG 2.0 Note](https://www.w3.org/TR/UAAG20/)
-
-## Technical Specifications
+## Especificaciones técnicas
 
 ### Accessible Rich Internet Applications (WAI-ARIA) {#aria}
 
-ARIA provides semantics so authors can convey user interface behaviors and structural information to assistive technologies (such as screen readers). The ARIA specification provides an ontology of roles, states, and properties that define accessible user interface elements.
+ARIA proporciona semántica, de tal forma que los autores pueden transmitir comportamientos de la interfaz de usuario e información estructural a las tecnologías de apoyo (por ejemplo, lectores de pantalla). La especificación de ARIA establece una ontología de roles, estados y propiedades que definen elementos accesibles en la interfaz de usuario.
 
-The ARIA suite includes <abbr title="application programming interface">API</abbr> mapping specifications that provide user agent implementation guidance. It also include modules for Graphics and Digital Publishing.
+La suite de ARIA incluye especificaciones de esquemas <abbr title="application programming interface">API</abbr> que proporcionan orientación para la aplicación en agentes de usuario. También incluye módulos para gráficos y la publicación digital.
 
-ARIA info:
-- [WAI-ARIA Overview](/standards-guidelines/aria/) – includes a [list and description of modules and API mappings](/standards-guidelines/aria/#versions)
-- [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices/)
-- [WAI-ARIA 1.1 Standard](https://www.w3.org/TR/wai-aria-1.1/)
+Información sobre ARIA:
+- [Resumen de WAI-ARIA](/standards-guidelines/aria/) – incluye [lista y descripción de módulos y esquemas API](/standards-guidelines/aria/#versions)
+- [Prácticas de autor de WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/)
+- [Estándar WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/)
 
-### Audio and Video {#multimedia}
+### Audio y vídeo {#multimedia}
 
-- [WebVTT: The Web Video Text Tracks Format](https://www.w3.org/TR/webvtt/) is a format for captions, text video descriptions, and other metadata that is time-aligned with audio or video content.
+- [Web Video Text Tracks (WebVTT)](https://www.w3.org/TR/webvtt/) es un formato para subtítulos, descripciones textuales de vídeo y otros metadatos que se sincronizan temporalmente con el contenido auditivo o visual.
```

Comment:
Same as before about translated and not translated the expanded version of the acronyms. For preventing more repetitions of this topic, this is the last time I mention, although there are a few more cases that in my opinion could be mentioned.

Reply:
Proposal: `Pistas de Texto para Vídeo en la Web`